### ISSUE TITLE
[MoM] Tone down Inertial Barrier (and maybe Concentration in general)

### ISSUE THREAD

**Issue by Terrorforge** [CleverRaven/Cataclysm-DDA#75577]
### Is your feature request related to a problem? Please describe.

I'm currently in lategame MoM (Winter 1 at time of writing, Metaphysics 10 with all three Concentration proficiences) with a Pyrokinetic/Telekinetic/Biokinetic, and I've hit on a pretty interesting strategy. Nether Attunement boosts ongoing effects, but you only trigger backlash effects when activating powers, so it's actually pretty damn strong to just start your day by activating Torrential Channeling, casting all your best buffs, and deliberately maxing out your Attunement so you get fire immunity, +20 to your physical stats and a 50% attack speed discount for free. The obvious downside to this is that using any of your active spells, including all those convenient utility powers, becomes a bit of a gamble. This forces you to make choices about when it's worth risking a strategic Burst of Speed and/or encourages you to play around these restrictions and rely more on non-psionic tools, both of which are fun and interesting.

Unfortunately, Inertial Barrier takes the fun out of it. It's honestly too strong in its own right, but especially as part of a buff stack at 12 attunement it just removes the risk/reward element by making you functionally immune to physical damage while an activity suit mostly takes care of the rest. I don't have to consider the possible ramifications of using a power to get away from a hulk, because I can just facetank it. I don't need to bring any special tools to deal with UGVs, because I can just facetank it. I don't need to be scared of hordes, because in combination with something like Blazing Aura, I can mow them down _by_ facetanking them.

Obviously I understand that late-game characters, especially late-game psychics, are going to be powerful. But this is powerful in such a boring way. I don't even know if the rest of the buff stack strategy is good because I'm pretty sure I still could've beat that star-crowned hound to death purely on the strength of Inertial Barrier.

### Solution you would like.

I'd make Inertial Barrier the bulletproofing power. Have it only grant nominal bash/cut/pierce resist, if any. 0.2/level at most. Probably turn down the ballistic resist to something like 1/level as well. Currently it makes you immune to 5.56 even at relatively modest levels with no Nether Attunement, but at 1/level you'd need a good chunk of attunement or mundane ballistic armor to reach that level. I think this would give it more of an identity rather than feeling like a cheat code. You could use it at low attunement (~20 ballistic armor) to protect against small arms and especially the shrapnel from your own grenades, or as part of a high-Attunement buff stack to style on UGVs, but it wouldn't render you immune to 90% of enemies.

It also just feels kind of weird that it's at all possible to abuse Concentration and Nether Attunement the way it is, but I'm not sure how to change that in a way that doesn't just make Concentration powers feel bad. Maybe if there were more consequences for just being at high Nether Attunement for extended periods? Tanking Health seems like an obvious one, maybe even straight up taking damage. Even just occasionally firing minor effects like nosebleeds and stamina loss when you're at 10+ Attunement would help it _feel_ less free and more like your whole body is vibrating with Netherium energy.

### Additional context

By posting this I am fully expecting that the next time I boot up the game I will discover the one obvious problem Inertial Barrier can't solve and die a very embarassing death, but that's a sacrifice I'm willing to make.

**Comment by Standing-Storm** [comment]
> Have it only grant nominal bash/cut/pierce resist, if any. 0.2/level at most. Probably turn down the ballistic resist to something like 1/level as well. 

Doing this would be balancing Inertial Barrier around requiring high Nether Attunement, which isn't true of any other power. An intelligence 10 character with maxed Inertial Barrier and no Nether Attunement would reduce bash/cut/pierce by 2 and ballistic damage by 10. It wouldn't even protect from small arms fire.

I did realize while reading this, though, that the way I update Nether Attunement failed to account for "climb to 12, set buffs, never open your powers menu and mow through everything" since it's an `open_spellbook`, so setting a recurring EoC to update your Nether Attunement in closer to real time is necessary.

**Comment by Terrorforge** [comment]
Oh, I meant keep the base values but change the scaling, so the ballistic arnor would be 12 + 1/level, leaving you with like 4 bash and 20 ballistic. That's a nice little buff that'll blunt zombie attacks significantly, keep you completely safe from small-calibre fire from e.g. bandits and at least save you from instantly losing a limb to 5.56, even mostly protecting you in conjunction with light ballistic armor.

I'd be inclined to tune the melee armor even lower so it _is_ like 2 bash at base just because armor is busted and even having ~6-8 at moderate Attunement and 16 at max is still really, really good, but that might be a personal bugbear.

**Comment by Terrorforge** [comment]
Having played around a bit more with/without it, I've come to the conclusion that it's not so much a balance issue with Inertial Barrier so much as armor just being really, really good in general. It's already kind of a design problem that the damage ranges are so tight that if you make a piece of armor remotely good, combining it with any other piece of armor that's remotely good immediately jumps you ahead to "indestructible". That's what's happening here; I'm wearing chainmail, and combining it with either Intertial Barrier or Momentum Alteration (and I have to assume Hardened Skin, but I haven't tried) crosses that threshold where virtually nothing can touch me any more. I still think it would be an improvement to skew the values on Inertial Barrier a bit more just to give it some more flavor, but I don't think the boringness is a balance flaw with that particular ability, just with armor in general.

If there is a deeper MoM-specific issue, it's how easy it is to maintain your concentration at higher levels. There's plenty of things that can break your concentration, but defensive concentration abilities protect you from most of them, sometimes in unexpected ways. For example, I don't get winded because Inertial Barrier means I'm not in a hurry to kill enemies, which means I can fall back on my knife or even my fists to maximize stamin efficiency. Just about the only thing that scares me at the moment are feral telepaths, who can Stun me out of my impenetrable cocoon. I should be afraid of nulls et. al. as well, but I've just discovered that they have some real problems that I'm just gonna detail in a separate bug report.

Fwiw I did almost lose this character. It was to delving into a collapsed tower and repeatedly huffing poison gas due to repeatedly running out of gas mask filters. It took constant exposure, multiple mistakes and me not having the damn bionic yet, but I guess there are some thigns that can get you.

Not sure what I even want any more, but those are some thoughts I've had recently and I hope they are in some way useful.

**Comment by Standing-Storm** [comment]
Yeah, for all the complexity elsewhere, armor is just straight -damage integer effect, not even as complex as Fallout armor values (where there was percentage-based reduction and flat reduction, often comboed), so it's extremely easy even as a normal character to end up immune to unevolved zombie attack. 

This is part of why I tended to design monsters so that their bump attacks do basically nothing and they have various special attacks that carry most of their combat weight. 

> repeatedly huffing poison gas due to repeatedly running out of gas mask filters. 

There's a long list of effects that lower your concentration limit and force a concentration check, and poison is one of them! But yes, if you're careful about never being winded, never being boomered, etc., you are going to find it much easier to maintain concentration

(some things I want to lower concentration limit, like being in a portal storm, I can't do yet because math isn't hooked up to weather so I can't detect it in the jmath equation.
